pkg/cloudprovider/vsphere/route/config/config_ini_legacy_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright 2020 The Kubernetes Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package config
+
+import (
+	"testing"
+)
+
+/*
+	TODO:
+	When the INI based cloud-config is deprecated. This file should be deleted.
+*/
+
+func TestReadINIConfig(t *testing.T) {
+	contents := `
+[Route]
+router-path = /infra/tier-1s/test-router
+
+[NSXT]
+user = admin
+password = secret
+host = nsxt-server
```

Comment:
can we name this field `server` instead of `host`?

Reply:
`[NSXT]` session was introduced by LB change, I guess there will be an upgrade issue if we change to use `server`. Or we just introduce a new field `server`?

Reply:
Oh good point, lets just leave it as host for this PR then

Reply:
Sure, thanks